Gedeonir/DoctorsAppointment
# Authentication

### **What does this PR do**
This PR includes Authentication endpoints
### **How to test this PR**
- Clone this repo if it is first time
- run `git checkout apointmentsEndpoints`
- Configure .env(see .env.example)
- run `npm install`
- run `run dev`
### **What to expect**

1. User can log into their account
2. User can reset and change password
